pkg/app/server.go

```diff
@@ -3,22 +3,25 @@ package app
 import (
 	"fmt"
 	"github.com/go-chi/chi"
-	log "github.com/sirupsen/logrus"
+	"github.com/iden3/prover-server/pkg/log"
 	"net/http"
 )
 
+// Server instance of chi server
 type Server struct {
 	Routes chi.Router
 }
 
+// NewServer creates new instance of server with routes
 func NewServer(router chi.Router) *Server {
 	return &Server{
 		Routes: router,
 	}
 }
 
+// Run starts the server
 func (s *Server) Run(port int) {
-	log.Infof("Server starting on port %d", port)
+	log.Infow("Server started", "port", port)
 	err := http.ListenAndServe(fmt.Sprintf(":%d", port), s.Routes)
-	log.Fatal(err)
+	log.Panic("server stopped", err)
```

Comment:
From the zap dicumentation:
> Panic uses fmt.Sprint to construct and log a message, then panics.
Probably panicking is not needed here.

Reply:
fixed